### ISSUE TITLE
create comments schema

### ISSUE THREAD

**Issue by razzlepdx** [Web-Bakers/web-bakers#21]
This is mostly done but the following needs to be added:

-comments should be able to be added to individual ideas and individual projects
-they should reference the user who wrote the comment
-they should reference the idea/project

**Comment by jsjoeio** [comment]
It looks like this is already done? In models/comment.js, there's a schema already defined. What else needs to be completed before this is closed?

**Comment by razzlepdx** [comment]
You are right @jjprevite!  Looking over the schema, the only that strikes me as missing might be that comments can be left on individual ideas, or on individual projects.  Should we have a reference to the idea/project as well as the user that wrote the comment? Or will the idea/project schemas store their own comments?

**Comment by jsjoeio** [comment]
@razzlepdx Yeah I think that's a good idea! Those should also be included.I'll add that to the description.

I don't think the comments need to be in the idea/project schemas, do you? The only thing I can think of is the number of comments. 

What does @mrreubenjones think?

**Comment by reuben-jones** [comment]
Whoops! Sorry gang, I had a backlog of github email to check, so this one was left waiting.

I'm not an expert in this field, but from my understanding we only need to have the connection reference on one side of the relationship. To me, it seems more appropriate to have the reference live in the idea and project schemas. (Either we have the parent listed in the comment schema, or the comment ID in the project/idea schema)

As for number of comments, that can be automatically calculated without needing to include it in the schema? How often will we need to check comments numbers?
